tools/stimulus/functions-R.yaml

```diff
@@ -374,6 +374,10 @@ igraph_get_all_simple_paths:
       PP: get.all.simple.paths.pp
 
 igraph_get_k_shortest_paths:
+    PARAM_ORDER: graph, from, to, *, k, ...
```

Comment:
I wonder why we need to override this here. No strong opinion.

Reply:
I did this to be able to make `weights` optional. This function will often be used on unweighted graphs. Is there a better way to do this?

Reply:
It's fine, but is this order something to be considered upstream too? To me, this order seems reasonable for other interfaces too.

Reply:
Well, yes, but probably too late for that.  If we ever reorder arguments in all functions in the C core, this would made sense, but it's unlikely that'll happen. When we settled on the argument ordering, we forgot about the consequences for auto-generation. CC @ntamas

https://github.com/igraph/igraph/wiki/Guidelines-for-function-argument-ordering

Reply:
Interesting.

In R land, we have the attribute. R objects are immutable, it's easy to remove weights, or to set new weights, without changing the graph object. I wonder if we even need the `weights` argument.